keyboards/jj40/README.md

```diff
@@ -69,3 +69,16 @@ Windows sometimes doesn't recognize the jj40. The easiest way of flashing a new
 3. If you get an error such as "Resource Unavailable" when attemting to flash
    on Linux, you may want to compile and run `tools/usb_detach.c`. See `tools/README.md`
    for more info.
+
+## Recovery 
```

Comment:
Actually, I think it would be best to refer to this article: 
https://docs.qmk.fm/#/isp_flashing_guide

That covers this stuff, and with more options. As well as precompiled binaries to use as an ISP flasher.

Reply:
that's fine for the ATmega32U4, but using the instructions there as-is won't work for the jj40 which based on the Atmega32A. the wiring stuff is sort of common though... i could tweak what i wrote with that page in mind if that helps.

Reply:
Ah, okay, that makes sense.

It may be worth linking to the page anyways, since is has links for precompiled hex files that you can use. 

Additionally, it would be awesome to expand that page to include this information, as well.  But that would be for another PR.

Reply:
ok. i'll make some changes & resolve this conversation when done